libs/indibase/baseclient.cpp

```diff
@@ -854,20 +766,194 @@ int INDI::BaseClient::messageCmd(XMLEle *root, char *errmsg)
             snprintf(msgBuffer, MAXRBUF, "%s: %s", ts, valuXMLAtt(message));
         }
 
-        std::string finalMsg = msgBuffer;
-
-        newUniversalMessage(finalMsg);
+        IDLog("%s\n", msgBuffer);
```

Comment:
Why is IDLog needed here? Any output to stderr could be detrimental to performance.